docs/features/pause.md

```diff
@@ -0,0 +1,43 @@
+# Pause KCC
+
+> `Feature State`: KCC version v1.114+ (alpha)
```

Comment:
I don't have a better suggestion, but this was a little hard to read.  Maybe "alpha as of version v1.114" ?  We could also ask taryn :-)